targets/PythonSdk/source/setup.py

```diff
@@ -0,0 +1,20 @@
+import setuptools
+with open("README.md", "r") as fh:
+        long_description = fh.read()
+        
+setuptools.setup(
+    name="playfab",
+    version="0.0.1",
```

Comment:
this needs to be a template for sdkVersion
Rename file, append ".ejs" to the end, and replace "0.0.1" with "<- sdkVersion >"
And add
sdkVersion = exports.sdkVersion
to the locals dict for the call to templatizeTree